source/Octopus.Cli/Commands/CreateReleaseCommand.cs

```diff
@@ -1,217 +1,262 @@
+
+            Log.Debug("Finding project: " + ProjectName);
+            var project = Repository.Projects.FindByName(ProjectName);
+            if (project == null)
+                throw new CouldNotFindException("a project named", ProjectName);
+
+            var plan = BuildReleasePlan(project);
+
+            string versionNumber;
+            if (!string.IsNullOrWhiteSpace(VersionNumber))
+            {
+                versionNumber = VersionNumber;
+                Log.Debug("Using version number provided on command-line: " + versionNumber);
+            }
+            else if (!string.IsNullOrWhiteSpace(plan.ReleaseTemplate.NextVersionIncrement))
+            {
+                versionNumber = plan.ReleaseTemplate.NextVersionIncrement;
+                Log.Debug("Using version number from release template: " + versionNumber);
+            }
+            else if (!string.IsNullOrWhiteSpace(plan.ReleaseTemplate.VersioningPackageStepName))
+            {
+                versionNumber = plan.GetActionVersionNumber(plan.ReleaseTemplate.VersioningPackageStepName);
+                Log.Debug("Using version number from package step: " + versionNumber);
+            }
+            else
+            {
+                throw new CommandException("A version number was not specified and could not be automatically selected.");
+            }
+
+            if (plan.IsViableReleasePlan())
+            {
+                Log.Info($"Release plan for {ProjectName} {versionNumber}{Environment.NewLine}{plan.FormatAsTable()}");
+            }
+            else
+            {
+                Log.Error($"Release plan for {ProjectName} {versionNumber}{Environment.NewLine}{plan.FormatAsTable()}");
+            }
+
+            if (plan.HasUnresolvedSteps())
+            {
+                throw new CommandException("Package versions could not be resolved for one or more of the package steps in this release. See the errors above for details. Either ensure the latest version of the package can be automatically resolved, or set the version to use specifically by using the --package argument.");
+            }
+
+            if (plan.HasStepsViolatingChannelVersionRules())
+            {
+                if (Force)
+                {
+                    Log.Warn($"At least one step violates the package version rules for the Channel '{plan.Channel.Name}'. Forcing the release to be created ignoring these rules...");
+                }
+
+                throw new CommandException($"At least one step violates the package version rules for the Channel '{plan.Channel.Name}'. Either correct the package versions for this release, select a different channel using --channel=MyChannel argument, let Octopus select the best channel using the --autoChannel argument, or ignore these version rules altogether by using the --ignoreChannelRules argument.");
+            }
+
+            if (IgnoreIfAlreadyExists)

```

Comment:
Just raising this one again... does this logic make sense?  

if `IgnoreIfAlreadyExists == true` and we do find an existing release, we exit without doing anything.  Shouldn't it be continuing on and replacing that release?